galloc/docs/optimization.md

```diff
@@ -0,0 +1,204 @@
+# Galloc implementations comparison
+
+1. [Summary](#summary)
+1. [Benchmark](#benchmark)
+1. [Current implementation](#current)
+1. [Implementation without static buffer](#without)
+1. [Implementation with increased by two static buffer](#increased-twice)
+1. [Implementation with multiple static buffers](#multiple)
+
+## Content
```

Comment:
When I wrote `content` I mean `Contents` like in the books with references:
```
Contents
- Summary
- Benchmark
- ...
```
Also it must be at the same level with other titles:
```
## Contents
- Summary
- Benchmark
- ....

## Summary
...
## Benchmark
... 
```